sources/engine/Xenko.VirtualReality/OpenVR/OpenVR.cs

```diff
@@ -1,18 +1,23 @@
 // Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
 // Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
-#if XENKO_GRAPHICS_API_DIRECT3D11
+#if XENKO_PLATFORM_WINDOWS_DESKTOP
```

Comment:
If enabling OpenVR on Windows in general, does OpenGL/OpenGLES still compiles?
Also, I think D3D11 was covering UWP too before (not sure if OpenVR was working on that platform either, if not it's OK for it to not be active anymore)

Reply:
Hrm.. I get no build errors, although I'm unsure if OpenGL/OpenGLES is trying to build...

Reply:
Various builds are broken.
https://teamcity.xenko.com/project.html?projectId=Engine&tab=projectOverview&branch_Engine=pull%2F407%2Fmerge&guest=1

Does it work on Vulkan Windows and Linux/OSX too?
if yes, the proper define would be `XENKO_GRAPHICS_API_DIRECT3D11 || XENKO_GRAPHICS_API_VULKAN`